nautobot/extras/tables.py

```diff
@@ -705,8 +705,9 @@ class JobResultTable(BaseTable):
                     <i class="mdi mdi-repeat"></i>
                 </a>
             {% else %}
-                <a href="#" class="btn btn-xs btn-default disabled" title="No saved job arguments, cannot be re-run">
-                    <i class="mdi mdi-repeat-off"></i>
+                <a href="{% url 'extras:job_run' slug=record.job_model.slug %}" class="btn btn-primary btn-xs"
```

Comment:
We have some inconsistency in button colors here (btn-success, btn-primary) and in the detail view (btn-info). Not sure the reasoning behind this?